app.rb

```diff
@@ -5,4 +5,18 @@
 
 get '/cars' do
   @cars = Car.all
-end
\ No newline at end of file
+  erb :index
+end
+
+get 'cars/:id' do
+  @cars = Car.find(params[:id])
+end
+
+get '/cars/new' do
+  erb :index
+end
```

Comment:
Since you added a form to your index, you don't necessarily need this route anymore.